tests/transactional/transactional_update.pm

```diff
@@ -38,9 +38,10 @@ sub get_utt_packages {
 
 # Check that package presence & version is as expected
 sub check_package {
-    my $stage   = shift // 'no';
+    my (%args) = @_;
+    my $stage   = $args{stage}   // 'no';
```

Comment:
Could you please describe possible values and their meaning somewhere, e.g. as perldoc for the method?

Reply:
added.